cmd/run.go

```diff
@@ -72,6 +75,24 @@ func start(cliCtx *cli.Context) error {
 	}
 	checkStateMigrations(c.StateDB)
 
+	// Prepare event log
+	var eventLog *event.EventLog
+	var eventStorage event.Storage
+
+	if c.EventLog.DB.Name != "disabled" {
```

Comment:
What happens if the EventLog.DB entry doesn't exists in the config file? if it doesn't exists we need to manage it in the same way if it have the "disabled" value. For non-trusted nodes (that will not use the event table) it will be more easy to avoid the creation of the [EventLog] [EventLog.DB] sections in the config file, it will be more transparent.

Reply:
This disabled literal will be read from the default.go so it does not need to be in the config toml files